src/odemis/acq/test/stream_test.py

```diff
@@ -3383,12 +3386,16 @@ def on_im(im):
             self.fail("Failed to find a second point in AR")
 
         e.wait()
+
+        time.sleep(5)  # TODO e.wait() seems to be not to be enough...why?? And why so long?
         im2d1 = ars.image.value
         # Check it's a RGB DataArray
         self.assertEqual(im2d1.shape[2], 3)
-
         self.assertFalse(im2d0 is im2d1)
 
+        with self.assertRaises(AssertionError):
+            numpy.testing.assert_array_equal(im2d0, im2d1)
```

Comment:
This feels a little odd to read. How about:
`assert not numpy.array_equal(im2d0, im2d1)`